src/main/java/frc/robot/RobotContainer.java

```diff
@@ -185,8 +182,12 @@ private void configureButtonBindings() {
         m_intake));
 
     // Vision align
-    ControlContants.driverX.whileTrue(new TapeAlign(m_robotDrive,m_Vision));
-
+    ControlContants.driverX.whileTrue(new TapeAlign(
+        m_robotDrive,
+        m_Vision,
+        () -> ControlContants.driverController.getRawAxis(0),
```

Comment:
Would probably change this to constants, similar to how it is where we set default command for drivebase